audit-service/src/contract-tests/resources/application.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020 Crown Copyright
```

Comment:
It looks like the one difference between this test application.yaml file and the one used for the production is the port.  If we are testing the system's ability to read this yaml file, this make sense.  If not it seems overkill.

Reply:
removed, wasnt needed

Reply:
This was one of the difficulties of the integration-tests repo, agreed not needed now the tests are in the same maven module